src/ensembl/production/metadata/api/genome.py

```diff
@@ -146,7 +146,11 @@ def fetch_genomes(self, genome_id=None, genome_uuid=None, organism_uuid=None, as
 
         # Apply additional filters based on the provided parameters
         if unreleased_only:
```

Comment:
imho, this parameter could be deducted from the `release` parameter, if set then filter by release, if not, then return unreleased. I am not sure this fit with the parameter name either: `unrelease_only` then you change to  `~Genome.genome_releases.any()` ? Am I right?

Reply:
> imho, this parameter could be deducted from the release parameter, if set then filter by release, if not, then return unreleased. I am not sure this fit with the parameter name either: unrelease_only then you change to ~Genome.genome_releases.any() ? Am I right?

The way the logic is currently implemented is that: 
* if `release` is not specified and `unreleased_only` is `Flase` (by default) -> the code should fetch Genome with `is_current=True` flag
  ```
  genome_select = genome_select.filter(GenomeRelease.is_current == 1
  ```
* if `release` is specified
  ```
  genome_select = genome_select.filter(EnsemblRelease.version <= release_version)
  ```
* if `unreleased_only==True`, we fetch unreleased genomes only
  ```
  genome_select = genome_select.filter(~Genome.genome_releases.any())
  ```

Does this make sense/answer your question?

Reply:
More or less, thank you for the explanation, I still struggle with the last one, why would Compara want to fetch unreleased "only", meaning they won't retrieve `any` released one....

Reply:
Something to ask to @ens-sb then! So we'll have the requirement right :-)

Reply:
He approved the idea, I'll implement it

Reply:
so now, I agree with the meaning of the param e.g. retrieve `unreleased data as well as the released one`. Once again, in the long run, I don't think @ens-sb will need to fetch unreleased data. 
Good luck with the implementation!

Reply:
@marcoooo I implemented the changes to accommodate all use cases